pandas/tests/io/formats/test_to_excel.py

```diff
@@ -169,11 +169,13 @@
      {'alignment': {'wrap_text': True}}),
 ])
 def test_css_to_excel(css, expected):
+    pytest.importorskip('cssdecl')
```

Comment:
the main thing to test is what happens when cssdecl is not installed and to_excel is called
need an informative error message